Chinese_Remainder_Theorem/Chinese_Remainder_Theorem.js

```diff
@@ -0,0 +1,59 @@
+/*
+The Chinese Remainder Theorem or CRT as it is commonly known is used to evaluate the smallest number 'n' such that:
+n % numbers[0] = remainder[0],
+n % numbers[1] = remainder[1],
+..............................
+n % numbers[length-1] = remainder[length-1].
+CRT belongs to the class of residue number system.
+CRT is given by relation n=Σ{s(i)*(remainders(i)*s1_inverse(i))mod divisors(i)}mod M, where:
+1. s(i) is partial product.
+2. s(i)_inverse is the inverse for partial product.
+3. remianders(i) is the remiander set value.
+4. divisors(i) is the divisor set value.
+5. M is known as the dynamic range
+*/
+
+//the inverse element is defined by (s1*s1_inverse)%moduli =1, where s1_inverse needs to be evaluated.
+const inverseElement = (s1, moduli) => {
+    let x_primary = 0;
+    let x_secondary = 1;
+    let moduli_primary = moduli;
+    if (moduli === 1) return 0;
+    while (s1 > 1) {
+        let quotient = Math.floor(s1 / moduli);
+        let temp = moduli;
+        moduli = s1 % moduli;
+        s1 = temp;
+        temp = x_primary;
+        x_primary = x_secondary - quotient * x_primary;
+        x_secondary = temp;
+    }
+    if (x_secondary < 0) {
+        x_secondary += moduli_primary;
+    }
+    return x_secondary;
+};
+
+const CRT = (divisors, remainders) => { //take the divisors and remainder set as arrays
+    const lengthDivisors = divisors.length;
+    const lengthRemainders = remainders.length;
+    if (lengthRemainders !== lengthDivisors) {
+        //if they dont match return since result cannot be evaluated
+        return `The length of remainder set and moduli set don't match,result cannot be evaluated`;
+    }
+    let M = 1; //M is the dynamic range evaluated as the product of all divisors.
```

Comment:
move this commend into previous line n also keep a blank line after if statement

Reply:
Done with this change